File: GGAndroid_2.0/GGDemo/src/main/java/io/ganguo/demo/view/activity/DialogDemoActivity.java
package io.ganguo.demo.view.activity;

import android.content.Context;
import android.content.Intent;

import io.ganguo.demo.view.activity.base.GGHFRVModelActivity;
import io.ganguo.demo.viewmodel.activity.DialogDemoVModel;
import io.ganguo.viewmodel.databinding.IncludeHfRecyclerBinding;
import io.ganguo.vmodel.ViewModelActivity;

/**
 * ViewModelPopupWindow 弹窗Demo
 * Created by leo on 2018/8/2.
 */

public class DialogDemoActivity extends GGHFRVModelActivity<IncludeHfRecyclerBinding, DialogDemoVModel> {
    public static Intent intentFor(Context context) {
        return new Intent(context, DialogDemoActivity.class);
    }

    @Override
    public DialogDemoVModel createViewModel() {
        return new DialogDemoVModel();
    }

    @Override
    public void onViewAttached(DialogDemoVModel viewModel) {
        super.onViewAttached(viewModel);
    }
}

File: GGAndroid_3.0/main/app-mvvm-v1-sample/src/main/java/io/ganguo/demo/view/activity/DialogDemoActivity.kt
package io.ganguo.demo.view.activity

import android.content.Context
import android.content.Intent

import io.ganguo.demo.view.activity.base.GGHFRVModelActivity
import io.ganguo.demo.viewmodel.activity.DialogDemoVModel
import io.ganguo.viewmodel.core.BaseViewModel
import io.ganguo.viewmodel.databinding.IncludeHfRecyclerBinding

/**
 * ViewModelPopupWindow 弹窗Demo
 * Created by leo on 2018/8/2.
 */

class DialogDemoActivity : GGHFRVModelActivity<IncludeHfRecyclerBinding, DialogDemoVModel>() {

    override fun createViewModel(): DialogDemoVModel {
        return DialogDemoVModel()
    }


    companion object {
        fun intentFor(context: Context): Intent {
            return Intent(context, DialogDemoActivity::class.java)
        }
    }

    override fun onViewAttached(viewModel: DialogDemoVModel) {
    }
}
